test/benchmark/benchmark.html

```diff
@@ -15,40 +16,42 @@
       margin: 0;
       padding: 0;
     }
+
     #map {
       position: absolute;
       height: 100%;
       width: 100%;
     }
   </style>
 </head>
+
 <body>
   <div id="map"></div>
   <script>
     const map = new mapboxgl.Map({
       container: 'map',
       style: 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json',
-      center: [0, 30],
-      zoom: 1
+      center: [-104.93637263263889, 39.72562104533162],
+      zoom: 11
     });
 
     carto.setDefaultAuth({
-      user: 'cartovl',
+      user: 'mamataakella',
```

Comment:
Why not copying the dataset to cartovl? (we better have a single account dependency for all things CARTO VL)

Reply:
Ups, I forgot